lib/src/app/ml_training.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:viam_sdk/protos/app/ml_training.dart';
+
+/// gRPC client for the [MlTrainingClient]. Used for working with ML training jobs.
```

Comment:
This is the `MLTrainingClient` so this docstring is recursive lol. Should this be for the `ML Training Service`?